bentoml/cli/bento_service.py

```diff
@@ -386,73 +378,22 @@ def containerize(bento, push, tag, build_arg, username, password, yatai_url):
         _echo(f"Found Bento: {saved_bundle_path}")
 
         bento_metadata = load_bento_service_metadata(saved_bundle_path)
-        name = to_valid_docker_image_name(bento_metadata.name)
-        version = to_valid_docker_image_version(bento_metadata.version)
-
-        if not tag:
-            _echo(
-                "Tag not specified, using tag parsed from "
-                f"BentoService: '{name}:{version}'"
-            )
-            tag = f"{name}:{version}"
-        if ":" not in tag:
-            _echo(
-                "Image version not specified, using version parsed "
-                f"from BentoService: '{version}'",
-                CLI_COLOR_WARNING,
-            )
-            tag = f"{tag}:{version}"
-
+        bento_tag = f'{bento_metadata.name}:{bento_metadata.version}'
+        yatai_client = get_yatai_client(yatai_url)
         docker_build_args = {}
         if build_arg:
             for arg in build_arg:
                 key, value = arg.split("=")
                 docker_build_args[key] = value
-
-        import docker
-
-        docker_api = docker.from_env().api
-        try:
-            with Spinner(f"Building Docker image {tag} from {bento} \n"):
-                for line in echo_docker_api_result(
-                    docker_api.build(
-                        path=saved_bundle_path,
-                        tag=tag,
-                        decode=True,
-                        buildargs=docker_build_args,
-                    )
-                ):
-                    _echo(line)
-        except docker.errors.APIError as error:
-            raise CLIException(f'Could not build Docker image: {error}')
-
-        _echo(
-            f'Finished building {tag} from {bento}', CLI_COLOR_SUCCESS,
-        )
-
-        if push:
-            auth_config_payload = (
-                {"username": username, "password": password}
-                if username or password
-                else None
+        with Spinner("Building Docker image"):
```

Comment:
Change the spinner message to something like this? ```Sending containerize RPC to YataiService {yatai_client.url}``` and for local yatai service use:  ```Containerizing {bento_tag} with local YataiService and docker deamon from env {docker_api.deamon_url}```

Reply:
Actually, would it make more sense to move this part to the Python API instead? that way users got to see the message and spinner even when using the Python API from a jupyter notebook

Reply:
As discussed offline. Spinner is better kept in the CLI client, because of it's implementation.